frontend_app_gallery/src/container/Gallery.jsx

```diff
@@ -631,9 +631,12 @@ export class Gallery extends React.Component {
   }
 
   displayReactImageLightBoxArrows (display) {
-    document.getElementsByClassName('ril__navButtons').forEach(arrow => {
-      if (arrow.style) arrow.style.visibility = display ? 'visible' : 'hidden'
-    })
+    const navButtons = document.getElementsByClassName('ril__navButtons')
+    for (let i = 0; i < navButtons.length; i++) {
+      const navButton = navButtons[i]
+      if (navButton.style)
+        navButton.style.visibility = display ? 'visible' : 'hidden'
```

Comment:
I know that you do not touch this code, but it seems wrong
```suggestion
        navButton.style.visibility === display ? 'visible' : 'hidden'
```

Reply:
I let you decide if you wanna fix or not

Reply:
Correct me if I'm wrong, but it's not a comparison here, is it?
The comparison is if display is true, then navButton.style.visibility is equal to visible, hidden overwise